dhash.py

```diff
@@ -33,26 +34,55 @@
 IS_PY3 = sys.version_info.major >= 3
 
 
-def get_grays(image, width, height):
+def get_grays_pil(image, width, height, fill_color='white'):
```

Comment:
Can we please rename these to `_get_grays_pil` and `_get_grays_wand` as I'd prefer they weren't part of the public API?